packages/apps/graph/src/services/chainweb-node/mempool.ts

```diff
@@ -0,0 +1,140 @@
+import type { Signer } from '@prisma/client';
+import { chainIds } from '@utils/chains';
+import { dotenv } from '@utils/dotenv';
+import https from 'https';
+
+export class MempoolError extends Error {
+  public mempoolError: any;
+
+  constructor(message: string, mempoolError?: any) {
+    super(message);
+    this.mempoolError = mempoolError;
+  }
+}
+
+export async function mempoolGetPending() {
+  return new Promise((resolve, reject) => {
+    const options = {
+      hostname: dotenv.MEMPOOL_HOSTNAME,
+      port: dotenv.MEMPOOL_PORT,
+      path: '/chainweb/0.0/development/chain/1/mempool/getPending',
+      method: 'POST',
+      rejectUnauthorized: false, // This disables certificate verification
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
```

Comment:
This fixes it for now. We disable certificate verification for this specific request. Will create a ticket to discuss the possibility of implementing an optional certificate integration